### ISSUE TITLE
Inserção de "Cards" na aba "Destaques"

### ISSUE THREAD

**Issue by matheus-nantes** [antonioThomaz1903/GCS_Issues_2024_1#7]
Na página inicial do site da FACOM existe um bloco chamado "Destaques". Ele possui elementos cuja formatação já é bem semelhante à "cards", formato amplamente utilizado atualmente em páginas webs.  Sugiro a troca dos elementos atuais para o formato de "cards" de modo que a interface fique mais intuitiva e agradável, pois os usuários tendem a clicar na imagem, por ser mais sugestiva e ter maior área. Porém, a única área clicável é o texto.
![Image](https://github.com/antonioThomaz1903/GCS_Issues_2024_1/assets/74730431/786c749e-12e8-4e53-adb4-97723b7453f6)

@antonioThomaz1903  e @LucasUlbrecht, o que acham desta sugestão?

**Comment by matheus-nantes** [comment]
Se tiverem alguma sugestão ou opinião diferente, discorra sobre, por favor!

**Comment by LucasUlbrecht** [comment]
Concordo totalmente com a sua sugestão, acredito que realmente parte do design daria a entender que os ícones em si funcionam como um objeto para clicar. Além disso, o que acham de remover o fundo azul ao redor dos ícones, e mudar a cor do ícone para azul, afim de modernizar e simplificar a forma dos ícones dentro dos Destaques, para ter uma referência visual mais precisa, vou enviar aqui um print do site da USP, que aplica esse ícones.
![imagem_2024-03-31_155442718](https://github.com/antonioThomaz1903/GCS_Issues_2024_1/assets/98244995/6bb59056-b66c-42bd-b316-2fb3d07107dd)
